src/App.js

```diff
@@ -1,12 +1,12 @@
 import { Container } from "./common/Container/styled";
-import Navigation from "./common/Navigation";
+import Header from "./common/Header";
 
 function App() {
   return (
     <>
-      <Navigation>
+      <Header>
 
-      </Navigation>
+      </Header>
       <Container>
 
       </Container>;
```

Comment:
if there is nothing in the component, it is better to write it out with a closing tag e.g. `<Navigation /> <Container />`